jekyll/_cci2/faq.md

```diff
@@ -54,20 +54,18 @@ Start with the [Hello World doc]({{ site.baseurl }}/2.0/hello-world/), then add
 
 ## Does CircleCI 2.0 run inference commands?
 
-Currently, CircleCI 2.0 doesn't infer anything from your project, but we have plans to replace our inference system before moving out of Beta. Until then, you'll need to manually configure all jobs.
+CircleCI 2.0 does not infer anything from your project and you must configure all jobs in the `config.yml` file.
```

Comment:
We will have some level of "inference" 

The inference will be a commented out YAML file. @rishimkumar can fill in the details here.